src/ActionParser.h

```diff
@@ -20,543 +20,603 @@
 
 namespace snowcrash {
 
-    /** Nameless action matching regex */
-    const char* const ActionHeaderRegex = "^[[:blank:]]*" HTTP_REQUEST_METHOD "[[:blank:]]*" URI_TEMPLATE "?$";
-
-    /** Named action matching regex */
-    const char* const NamedActionHeaderRegex = "^[[:blank:]]*" ACTION_SYMBOL_IDENTIFIER "\\[" HTTP_REQUEST_METHOD "[[:blank:]]*" URI_TEMPLATE "?]$";
-
-    /** Miss leading slash in URI */
-    const char* const NamedActionNonAbsoluteURIRegex = "^[[:blank:]]*" ACTION_SYMBOL_IDENTIFIER "\\[" HTTP_REQUEST_METHOD "[[:blank:]]+[^/]+]$";
-
-    /** Internal type alias for Collection iterator of Action */
-    typedef Collection<Action>::const_iterator ActionIterator;
-
-    /** Action Definition Type */
-    enum ActionType {
-        NotActionType = 0,
-        DependentActionType,      /// Action isn't fully defined, depends on parents resource URI
-        CompleteActionType,       /// Action is fully defined including its URI
-        UndefinedActionType = -1
-    };
-
-    /**
-     * Action Section processor
-     */
-    template<>
-    struct SectionProcessor<Action> : public SectionProcessorBase<Action> {
+/** Nameless action matching regex */
+const char *const ActionHeaderRegex =
+    "^[[:blank:]]*" HTTP_REQUEST_METHOD "[[:blank:]]*" URI_TEMPLATE "?$";
+
+/** Named action matching regex */
+const char *const NamedActionHeaderRegex =
+    "^[[:blank:]]*" ACTION_SYMBOL_IDENTIFIER "\\[" HTTP_REQUEST_METHOD
+    "[[:blank:]]*" URI_TEMPLATE "?]$";
+
+/** Miss leading slash in URI */
+const char *const NamedActionNonAbsoluteURIRegex =
+    "^[[:blank:]]*" ACTION_SYMBOL_IDENTIFIER "\\[" HTTP_REQUEST_METHOD
+    "[[:blank:]]+[^/]+]$";
+
+/** Internal type alias for Collection iterator of Action */
+typedef Collection<Action>::const_iterator ActionIterator;
+
+/** Action Definition Type */
+enum ActionType
+{
+    NotActionType = 0,
+    DependentActionType, /// Action isn't fully defined, depends on parents
+                         /// resource URI
+    CompleteActionType,  /// Action is fully defined including its URI
+    UndefinedActionType = -1
+};
+
+/**
+ * Action Section processor
+ */
+template <>
+struct SectionProcessor<Action> : public SectionProcessorBase<Action>
+{
+
+    static MarkdownNodeIterator processSignature(
+        const MarkdownNodeIterator &node,
+        const MarkdownNodes &siblings,
+        SectionParserData &pd,
+        SectionLayout &layout,
+        const ParseResultRef<Action> &out) {
+
+        actionHTTPMethodAndName(
+            node, out.node.method, out.node.name, out.node.uriTemplate);
```

Comment:
This is strange, it is possible to break line before param which exceed column limit?